### PR TITLE
WIP: feat(contributing-docs): Add Code of Conduct, Contributing.md and Issue/Pull Request templates

### PR DESCRIPTION
**Pull request by nikazawila**
This PR is meant to add edited copy for Code of Conduct, Contributing.md, Issue/Pull Requests templates and Support.md

### CODE REVIEW

**Comment by Rendez** [comment]
Is CLA going to be added with https://github.com/cla-assistant/cla-assistant ?

**Comment by nikazawila** [comment]
I guess we first need an approved version of the CLA before we decide however we want to deal with it?

**Comment by weaintplastic** [comment]
CLA assistant should be integrated. but probably a bigger topic. I'll add it to the backlog

**Review by Rendez — APPROVED** [review]

**Comment by nikazawila** [comment]
this is ready for a final review (mostly technical parts like Project setup in Contributing.md) - rest was already edited after Carrie's comments :)

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,142 @@
-# Contributing to the project
+# Contributing Guide
 
-The following is a set of guidelines for contributing to this repository on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+Thanks for taking the time to contribute!
+
+#### Table of contents
```
Comment (Rendez) [inline .github/CONTRIBUTING.md]: You can make this foldable, e.g.:

```
<details>
<summary><strong>Table of Contents</strong></summary>

* [What does IntersectionObserver do?](#what-does-intersectionobserver-do)
* [...]
```

**Comment by Rendez** [comment]
I personally don't think we need `SUPPORT.md`, github issues will be used for support

**Pull request merged**